config/trader/IEL_b1.py

```diff
@@ -276,19 +278,16 @@ def Vexperimentation(self, action):
         :param action: 0 corresponds to buying, 1 corresponds to selling
         """
         for j in range(self.J):
-            sigmav = 1
             if action == self.BUY:
-                if self.get_units() < self.max_holdings:
```

Comment:
We would need this condition when the bot has exactly the max holdings or greater to prevent index holdings, so I would suggest keeping it.

Reply:
This is related to the previous comment: it would not be a problem as long as the additional 0 in the valuation array is kept.